diff --git a/typescript/end-to-end-applications/food-ordering/app/src/order-app/order_workflow/impl.ts b/typescript/end-to-end-applications/food-ordering/app/src/order-app/order_workflow/impl.ts
--- a/typescript/end-to-end-applications/food-ordering/app/src/order-app/order_workflow/impl.ts
+++ b/typescript/end-to-end-applications/food-ordering/app/src/order-app/order_workflow/impl.ts
@@ -39,9 +39,3 @@ export default restate.workflow({
       const token = ctx.rand.uuidv4();
-      const paid = await ctx.run("payment", () => paymentClient.charge(id, token, totalCost));
-
-      if (!paid) {
-        ctx.set("status", Status.REJECTED);
-        return;
-      }
-
+
       // 3. Schedule preparation
